### PR TITLE
chore(*) bump Kong version

### PR DESCRIPTION
**Pull request by salazar**

### CODE REVIEW

File: library/kong [hunk 1]
```diff
@@ -2,12 +2,13 @@ Maintainers: Mashape Docker Maintainers <support@konghq.com> (@thekonginc)
...
 
-Tags: 0.12.0rc2-alpine
-GitCommit: 9f2d6a104dbf8b38cb8188429c17daef72f24196
+Tags: 0.12-alpine, 0.12.0-alpine
+GitCommit: 9a20698ccb907b1a9d473e2f41e7ccdbd7de8c99
 Directory: alpine
 
-Tags: 0.12.0rc2
-GitCommit: 9f2d6a104dbf8b38cb8188429c17daef72f24196
+Tags: 0.12-centos, 0.12.0-centos, 0.12, 0.12.0, latest
```
Comment (yosifkit) [inline library/kong]: This new `latest` is conflicting with the old `latest` on `0.11.2`.  Probably just drop latest from `0.11`?

Any particular reason to keep the old version listed below?  Removing it here will remove it from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding it on the official build server), but the tag will still be available to users who want it.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)

Comment (salazar) [reply library/kong]: @yosifkit, removed the extra `latest` tag - my apologies. On removing old versions, I will discuss it with other folks before removing - thanks for the feedback.

**Comment by yosifkit** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-arches b/_bashbrew-arches
index 8e96679..9fda995 100644
--- a/_bashbrew-arches
+++ b/_bashbrew-arches
@@ -1,6 +1,6 @@
 kong:0.9.9 @ amd64
 kong:0.10.4 @ amd64
+kong:0.11.2 @ amd64
 kong:0.11.2-alpine @ amd64
-kong:0.12.0rc2 @ amd64
-kong:0.12.0rc2-alpine @ amd64
+kong:0.12.0-alpine @ amd64
 kong:latest @ amd64
diff --git a/_bashbrew-list b/_bashbrew-list
index 468ec7b..02bea6f 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -6,6 +6,10 @@ kong:0.11
 kong:0.11-alpine
 kong:0.11.2
 kong:0.11.2-alpine
-kong:0.12.0rc2
-kong:0.12.0rc2-alpine
+kong:0.12
+kong:0.12-alpine
+kong:0.12-centos
+kong:0.12.0
+kong:0.12.0-alpine
+kong:0.12.0-centos
 kong:latest
diff --git a/kong_latest/Dockerfile b/kong_0.11.2/Dockerfile
similarity index 100%
copy from kong_latest/Dockerfile
copy to kong_0.11.2/Dockerfile
diff --git a/kong_0.12.0rc2-alpine/docker-entrypoint.sh b/kong_0.11.2/docker-entrypoint.sh
similarity index 100%
rename from kong_0.12.0rc2-alpine/docker-entrypoint.sh
rename to kong_0.11.2/docker-entrypoint.sh
diff --git a/kong_0.12.0rc2-alpine/Dockerfile b/kong_0.12.0-alpine/Dockerfile
similarity index 88%
rename from kong_0.12.0rc2-alpine/Dockerfile
rename to kong_0.12.0-alpine/Dockerfile
index 0e97197..b960881 100644
--- a/kong_0.12.0rc2-alpine/Dockerfile
+++ b/kong_0.12.0-alpine/Dockerfile
@@ -1,8 +1,8 @@
 FROM alpine:3.6
 LABEL maintainer Marco Palladino, marco@mashape.com
 
-ENV KONG_VERSION 0.12.0rc2
-ENV KONG_SHA256 d7ce17e5cf95422f278dcc1d3e6469e378bb6875b5e8676df8be2fb33daf28e4
+ENV KONG_VERSION 0.12.0
+ENV KONG_SHA256 43683a0cde80fb3a952e84d1c7748fc3e178a8902ce543b2baf5c7c65b7b52d9
 
 RUN apk add --no-cache --virtual .build-deps wget tar ca-certificates \
 	&& apk add --no-cache libgcc openssl pcre perl tzdata \
diff --git a/kong_0.12.0rc2/docker-entrypoint.sh b/kong_0.12.0-alpine/docker-entrypoint.sh
similarity index 100%
rename from kong_0.12.0rc2/docker-entrypoint.sh
rename to kong_0.12.0-alpine/docker-entrypoint.sh
diff --git a/kong_0.12.0rc2/Dockerfile b/kong_0.12.0rc2/Dockerfile
deleted file mode 100644
index cdc1311..0000000
diff --git a/kong_latest/Dockerfile b/kong_latest/Dockerfile
index 2160c5b..839e6e6 100644
--- a/kong_latest/Dockerfile
+++ b/kong_latest/Dockerfile
@@ -1,7 +1,7 @@
 FROM centos:7
 MAINTAINER Marco Palladino, marco@mashape.com
 
-ENV KONG_VERSION 0.11.2
+ENV KONG_VERSION 0.12.0
 
 RUN yum install -y wget https://bintray.com/kong/kong-community-edition-rpm/download_file?file_path=dists%2Fkong-community-edition-$KONG_VERSION.el7.noarch.rpm && \
     yum clean all
```

</details>

**Comment by yosifkit** [comment]
Build test of #3907; e520cd09; `amd64` (`kong`):

```console

$ bashbrew build kong:0.12-alpine
Building bashbrew/cache:f772f7271068e2cc359d8f4561b1afa661d9f7119316ac1b86f3ee11dfbf19b1 (kong:0.12-alpine)
Tagging kong:0.12-alpine
Tagging kong:0.12.0-alpine

$ test/run.sh kong:0.12-alpine
testing kong:0.12-alpine
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build kong:0.12-centos
Building bashbrew/cache:ad1971b728329264f331b9396dad6afa62b48b4c3f0c96bcbf62847c6392a9b2 (kong:0.12-centos)
Tagging kong:0.12-centos
Tagging kong:0.12.0-centos
Tagging kong:0.12
Tagging kong:0.12.0
Tagging kong:latest

$ test/run.sh kong:0.12-centos
testing kong:0.12-centos
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...warning: garbage password detected for 'root': 'locked'
passed
	'override-cmd' [4/4]...passed


$ bashbrew build kong:0.11-alpine
Using bashbrew/cache:3410c0cddd59c359de2ae062045f4be89cfa876a24ca3fc77c772958e2f55b4f (kong:0.11-alpine)
Tagging kong:0.11-alpine
Tagging kong:0.11.2-alpine

$ test/run.sh kong:0.11-alpine
testing kong:0.11-alpine
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build kong:0.11
Using bashbrew/cache:71adb8700281a32c8adda6e94da2b887ced85e95a5df84d17e14522ee55cfb52 (kong:0.11)
Tagging kong:0.11
Tagging kong:0.11.2

$ test/run.sh kong:0.11
testing kong:0.11
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...warning: garbage password detected for 'root': 'locked'
passed
	'override-cmd' [4/4]...passed


$ bashbrew build kong:0.10
Using bashbrew/cache:7e76d866fed5d3a29b56d3ad67986f43eeb47362473c5f1a43b8c05c79b68ba6 (kong:0.10)
Tagging kong:0.10
Tagging kong:0.10.4

$ test/run.sh kong:0.10
testing kong:0.10
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...warning: garbage password detected for 'root': 'locked'
passed
	'override-cmd' [4/4]...passed


$ bashbrew build kong:0.9
Using bashbrew/cache:f498c6b3f4137a2cf102574d7d64eda7a86e644969b6781d4bc6a771a11042db (kong:0.9)
Tagging kong:0.9
Tagging kong:0.9.9

$ test/run.sh kong:0.9
testing kong:0.9
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...warning: garbage password detected for 'root': 'locked'
passed
	'override-cmd' [4/4]...passed

```

**Pull request merged**